src/mumble/CustomElements.cpp

```diff
@@ -162,13 +183,35 @@ void ChatbarTextEdit::insertFromMimeData(const QMimeData *source) {
 
 			if (static_cast<unsigned int>(imgHtml.length()) < g.uiImageLength) {
 				emit pastedImage(imgHtml);
+				return true;
 			} else {
-				g.l->log(Log::Information, tr("Unable to send image: too large."));
+				g.l->log(Log::Information, tr("Unable to send image %1: too large.").arg(QLatin1String("")));
```

Comment:
This doesn't seem to make too much sense here. You're inserting an empty String...

Reply:
Because I don't want to increase the workload of translating... You can see, I've used the same template string a few lines earlier.

Reply:
I see what your idea is, but I think we shouldn't do it this way. Whoever reads this code will be wondering what black magi is going on, because "obviously nobody *just* wants to insert an empty String".

And the translation overhead is really low. I'm pretty sure Transifex will propose the same translation to be Copy&Pasted for this case as it recognizes that they are almost identical :)